src/utils/provider.ts

```diff
@@ -37,6 +43,25 @@ export function wait(delay: number) {
  * Create Sierra Contract Class from a given Compiled Sierra
  *
  * CompiledSierra -> SierraContractClass
+ * 
+ * @param {CompiledSierra} contract - Compiled sierra code
+ * 
+ * @example
+ * ```typescript
+ * const result = createSierraContractClass({
+    "sierra_program": [
+        "0x1",
+        "0x4",
+        "0x0",
+        "0x2",
+        "0x4",
+        "0x1",
+        "0x3b4",
+        "0x4c",
+        "0x65",
+        "0x52616e6765436865636b",...})
+ * // result = {sierra_program: 'H4sIAAAAAAAAA6x9WZbsrI7uVGqd53qgb8ZynwzYY7jDv5JAAmxHZuQ+96yq/L0jIzEINZ8axP/5j/q/+j//+z/wH9f/o/p/zPbh+Iot49+u9v8G3//rTdDhDDF4Z0MKPthQ+m+S2v6n1S//638VvdXW2PQ6RvxuDG+jiybCXKJ7Hef6ZRi9E+Q89WmKLilfqbrsL6PUCf8...}
+ * ```
```

Comment:
`createSierraContractClass` and `parseContract` are performing the opposite action of the other, but you have the same example and result for both cases.
You should re-test these examples ; you will quickly find your error.